etc/system-contracts/bootloader/test_infra/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "test_infra"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+vm = { git = "https://github.com/matter-labs/zksync-era.git", branch = "boojum-integration" }
```

Comment:
Could you please reference to the main branch. 
I see that vm crate is ultra outdated

Reply:
It was copied from the original system contracts, and it's outdated there. We will update it, but maybe with 1.4.1. However here this script will not be used, and actually, we can remove it.

Reply:
Tried to remove the test directories, but they're needed as part of the `etc/system-contracts/scripts/process.ts` file

Will leave as-is for now